src/pages/TransactionsList.vue

```diff
@@ -58,12 +61,51 @@
         </div>
       </template>
     </BaseTable>
+
+    <BaseTable
+      v-else
+      :loading="instructionsTable.loading.value"
+      :pagination="instructionsTable.pagination"
+      :items="instructionsTable.items.value"
+      container-class="transactions-list-page__container"
+      @next-page="instructionsTable.nextPage()"
+      @prev-page="instructionsTable.prevPage()"
+      @set-page="instructionsTable.setPage($event)"
+      @set-size="instructionsTable.setSize($event)"
+    >
+      <template #row="{ item }">
```

Comment:
Yeah. It's better to have exact payload types for every single instruction type (I have already left todo on instruction schema). Probably as soon as js sdk will have it - I will use it. At least it will be better to display exact data instead of whole json.

Reply:
I will add it to our todo list.

Reply:
Sure. Later we will also display more details especially values.